packages/atlas/src/providers/transactionManager/useTransaction.ts

```diff
@@ -100,6 +105,17 @@ export const useTransaction = (): HandleTransactionFn => {
           addBlockAction({ callback: syncCallback, targetBlock: result.block })
         })
 
+        const { data } = await getTransactionStatus({ variables: { transactionHash: result.transactionHash } })
```

Comment:
We should probably add some polling if the data was not found, but with some retry timeout, if possible. I would suggest poll every 1 sec for max 10 tries